src/coreclr/jit/emit.h

```diff
@@ -298,8 +298,9 @@ struct insGroup
 #if EMIT_TRACK_STACK_DEPTH
     unsigned igStkLvl; // stack level on entry
 #endif
-    regMaskSmall  igGCregs; // set of registers with live GC refs
-    unsigned char igInsCnt; // # of instructions  in this group
+    regMaskSmall  igGCregs;    // set of registers with live GC refs
+    unsigned char igInsCnt;    // # of instructions in this group
+    unsigned char igZeroSzCnt; // # of zero size instructions in this group
```

Comment:
It appears `igZeroSzCnt` is only used in DEBUG; can it and the things that feed it (`emitCurIGZeroSzCnt`) be only under `DEBUG`?

Reply:
Will update

Reply:
This looks to only be used under `EMITTER_STATS`, the same as everything else in the same group, and so should already be filtered correctly.

Reply:
Nevermind, I was looking at the wrong thing. This one is only used under `DEBUG` and so I've conditioned it appropriately